build/NuSpecs/WindowsAppSDK-Nuget-Native.targets

```diff
@@ -16,6 +16,12 @@
     </Reference>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(AppxPackage)' == 'true'">
```

Comment:
It looks like the owner of DynamicDependency.winmd wanted this to be included only when AppxPackage=='false'.  @DrusTheAxe, can you confirm that?  Thanks!

Reply:
Confirmed

DynamicDependency API is only supported in unpackaged processes

Reply:
Yes it looks like that, however I kept it for being on safe side by keeping existing behavior.

Reply:
Thanks @DrusTheAxe, Removed that entry

Reply:
DynDep explicitly doesn't support Packaged apps. All the APIs in [dev/DynamicDependency/API/MsixDynamicDependency.cpp](https://github.com/microsoft/WindowsAppSDK/blob/main/dev/DynamicDependency/API/MsixDynamicDependency.cpp) have

```
    // WinAppSDK's Dynamic Dependencies requires a non-packaged process
    RETURN_HR_IF(HRESULT_FROM_WIN32(ERROR_NOT_SUPPORTED), AppModel::Identity::IsPackagedProcess());
```